pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -569,9 +572,12 @@ type DatacenterSpecDigitalocean struct {
 
 // DatacenterSpecOpenstack describes an OpenStack datacenter.
 type DatacenterSpecOpenstack struct {
-	AuthURL          string `json:"authURL"`
+	// Authentication URL
+	AuthURL string `json:"authURL"`
+	// Used to make resources highly available
```

Comment:
Is it though? You have to select 1 AZ to put every VM into. Having multiple AZs might make your Openstack setup more reliable, but here for a datacenter config this is not "making clusters highly available".